src/app_service/service/contacts/service.nim

```diff
@@ -178,7 +191,7 @@ QtObject:
     result = self.fetchContact(id)
     if result.id.len == 0:
       let alias = self.generateAlias(id)
-      let identicon = self.generateIdenticon(id)
+      let identicon = self.imageServerUrl & identiconURLParam & id
```

Comment:
@richard-ramos if you don't mind, please revert this line. It's far easier if we have build identicon in a single place.

Reply:
I agree with Sale here. Actually, I feel like this should be handled by status-go (the same way the message image is handled), but it can be in another PR